### PR TITLE
Update README.md

### PR DESCRIPTION
**Pull request by KSchlobohm**

### CODE REVIEW

**Review by nickdala — APPROVED** [review]
LGTM

**Comment by stephen-sumner** [comment]
I think postgresql should have high availability on in primary. based on the terraform: https://github.com/stephen-sumner/reliable-web-app-pattern-java/blob/42c2728a47e0c45720e5949c6523a6773596f3d6/terraform/modules/postresql/main.tf#L57

**Comment by stephen-sumner** [comment]
@KSchlobohm - this link is currently live in AAC. Let's merge this PR today. And open another one if we need to update. I just want to have parity across AAC and the repo.

File: README.md [hunk 1]
```diff
@@ -22,8 +22,8 @@ For more information on the reliable web app pattern, see [Overview](https://rev
 
 [![Diagram showing the architecture of the reference implementation](docs/assets/reliable-web-app-java.svg)](docs/assets/reliable-web-app-java.svg#lightbox)
 
-- [Production environment estimated cost](https://azure.com/e/a44f5feb443f430abbd9116b6cc879bf)
-- [Non-production environment estimated cost](https://azure.com/e/807cdb0d6d9a41899885bd875deb12f5)
+- [Production environment estimated cost](https://azure.com/e/9ed08d9eb2ba426e95e4e7f5a8b075e7)
```
Comment (KSchlobohm) [inline README.md:25]:
```suggestion
- [Production environment estimated cost](https://azure.com/e/4e27d768a5924e3d93252eeceb4af4ad)
```

File: README.md [hunk 2]
```diff
@@ -22,8 +22,8 @@ For more information on the reliable web app pattern, see [Overview](https://rev
...
+- [Non-production environment estimated cost](https://azure.com/e/b6ac4fdb62f3475cb1c2dee388dc4b9a)
```
Comment (KSchlobohm) [inline README.md:26]:
```suggestion
- [Non-production environment estimated cost](https://azure.com/e/1721b2f3f2bd4340a00115e79057177a)
```
Apply Dev/Test pricing for non-prod and use smaller (50gB) storage estimate for non-prod.

**Pull request merged**